test/e2e/framework/auth/helpers.go

```diff
@@ -57,12 +57,12 @@ func WaitForNamedAuthorizationUpdate(c v1beta1authorization.SubjectAccessReviews
 			User: user,
 		},
 	}
+
 	err := wait.Poll(policyCachePollInterval, policyCachePollTimeout, func() (bool, error) {
 		response, err := c.SubjectAccessReviews().Create(review)
 		// GKE doesn't enable the SAR endpoint.  Without this endpoint, we cannot determine if the policy engine
 		// has adjusted as expected.  In this case, simply wait one second and hope it's up to date
-		if apierrors.IsNotFound(err) {
-			klog.Info("SubjectAccessReview endpoint is missing")
+		if apierrors.IsNotFound(err) && framework.ProviderIs("gke"){
```

Comment:
If we only document this err path as being for GKE lets test it as such...

Reply:
Unfortunately, this dependence on the `framework` was too hard to break so I removed this check and added a FIXME comment.

The `ProviderIs` depends on the `TestContext` which depends on the `Framework` object. Those are other, large items in the framework that we can consider breaking down/putting into their own packages but at this time it is not as feasible.

Removing that check isn't a regression at least, it is just returning the functionality to the way it was before.